modules/io/python/drivers/io/tests/test_ossfs.py

```diff
@@ -33,8 +33,7 @@
 import pytest
 import requests
 
-pytestmark = pytest.mark.skipif(True, reason='skip ossfs test')
-
+ossfs = pytest.importorskip("ossfs")
 from ossfs import OSSFileSystem
```

Comment:
The variable name conflicits with the package name...

Reply:
If import succedd, `ossfs` will be that module, just like `mod = importlib.import_module("mod")`